assets/admin/editor-wizard/use-sensei-pro-extension.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { EXTENSIONS_STORE } from '../../extensions/store';
+
+export default function useSenseiProExtension() {
```

Comment:
I created this hook based on [this component](https://github.com/Automattic/sensei/blob/a21c5162e31bc88079a76301fe6740aae2b6699d/assets/extensions/featured-product-sensei-pro.js) that I found inside the Sensei Core repository. I accept any feedbacks about how to improve this, as I need to get the extensions detail both on the Wizard component (to remove the Paid Course upsell if Sensei Pro extension is installed) as well as on the Course Upgrade Step (to get the price dynamically).

Reply:
Moved the discussion to [this thread](https://github.com/Automattic/sensei/pull/5169#pullrequestreview-983182993) instead (after moving the hook out of the editor-wizard folder)